src/TSTransformer/classes/TransformState.ts

```diff
@@ -49,6 +55,37 @@ export class TransformState {
 		this.sourceFileText = sourceFile.getFullText();
 	}
 
+	public readonly tryUsesStack = new Array<TryUses>();
+
+	/**
+	 * Pushes tryUses information onto the tryUses stack and returns it.
+	 */
+	public pushTryUsesStack() {
+		const tryUses = {
+			usesReturn: false,
+			usesBreak: false,
+			usesContinue: false,
+		};
+		this.tryUsesStack.push(tryUses);
+		return tryUses;
+	}
+
+	/**
+	 * Marks the current try statement as exiting with return, break, or continue statements.
+	 */
+	public markTryUses(property: "usesReturn" | "usesBreak" | "usesContinue") {
```

Comment:
This could be `property: keyof TryUses`

Reply:
My limited experience with typescript shows 😛 I'll fix this.